tests/test_acasclient.py

```diff
@@ -2544,6 +2546,24 @@ def test_051_bulk_load_update_parent_alias(self):
         self.assertIn(alias_two, aliases)
         self.assertIn(alias_three, aliases) 
 
+        # Redo the same file (copy) to see if the result is the same 
+        # and aliases aren't just being appended without comparison 
+        test_051_upload_file_file_one_copy = Path(__file__).resolve().parent\
```

Comment:
Is this literally the same file twice with different filenames? Why not just upload `test_051_register_parent_aliases.sdf` again?

Reply:
It is. I thought it was reasonable to replicate the circumstance where someone is trying to register the same compound but some of the file metadata information (e.g. data created, date modified, etc.) is slightly different. Not a big deal. I can re-register the same file.

Reply:
Ah, okay. Not a big deal. I was just wondering why. I don't think we track any of the file metadata information, but it's fine for you to keep this as-is. Consider this one truly optional.